### PR TITLE
icds pgucr tune 2

### PR DESCRIPTION
**Pull request by snopoke**
@dimagi/scale-team 
buddies @pr33thi @orangejenny

### CODE REVIEW

File: environments/icds-new/inventory.ini [hunk 1]
```diff
@@ -74,7 +74,8 @@ postgresql_checkpoint_completion_target=0.9
 postgresql_maintenance_work_mem = 2GB
 postgresql_wal_buffers=16MB
 postgresql_default_statistics_target = 500
-postgresql_max_wal_size = 1GB
+postgresql_max_wal_size = 5GB
```
Comment (snopoke) [inline environments/icds-new/inventory.ini]: See discussion here: https://github.com/dimagi/commcare-cloud/pull/1911/files#r209358599

File: environments/icds-new/inventory.ini [hunk 2]
```diff
@@ -74,7 +74,8 @@ postgresql_checkpoint_completion_target=0.9
...
+postgresql_random_page_cost = 2.0
```
Comment (snopoke) [inline environments/icds-new/inventory.ini]: See https://github.com/dimagi/commcare-cloud/pull/1707/files#r186975976

**Comment by snopoke** [comment]
@emord would you be able to deploy this?

**Comment by emord** [comment]
@snopoke I'll do it tomorrow

**Comment by snopoke** [comment]
👍

**Comment by emord** [comment]
Deployed. also not seeing anymore warnings about the wal_size

**Pull request merged**